ovos_tts_plugin_piper/__init__.py

```
@@ -141,20 +165,41 @@ def get_tts(self, sentence, wav_file, lang=None, voice=None, speaker=None):
+            engine, speaker, voice = self.lang2model(lang, voice, speaker)
+        elif lang:
+            # requested a language but not a voice
+            # - try to use default voice, but force language via phonemizer
+            # - change to a different TTS model completely
+            try:
+                # force a specific espeak phonemizer to match lang pronunciation
+                # this allows a voice to speak a different language a bit better
+                phonemizer_lang = get_espeak_voice(lang)
+                LOG.debug(f"Forcing Piper accent: {phonemizer_lang}")
+                engine, speaker, voice = self.lang2model(self.lang, self.voice)
+            except:  # change to a voice that supports the lang
```

Ruff (0.8.2)

Do not use bare `except`

(E722)